README.md

```diff
@@ -1,30 +1,39 @@
 # Vantaa Exercise Reservations
 
 ## Architecture
-This project is simple client-server application built with 
-- Frontend: React.js & MobX
-- Backend: Node.js & Express.js
-- Database: PostgreSQL
 
-It integrates to Grynos to get the information about current courses available in Vantaa.
+This project is simple client-server application built with
+
+*   Frontend: React.js & MobX
+*   Backend: Node.js & Express.js
+*   Database: PostgreSQL
 
+It integrates to Grynos to get the information about current courses available in Vantaa.
 
 ## Running locally
+
+The local development is set up using docker. Docker is a containerization that help shipping application easy, and without the hassle of installing many many things. We highly recommend you install docker.
```

Comment:
No need to explain what is Docker and why it is used. And don't also think that after these changes you can say it is recommended as it is basically the only way provided.

Reply:
To new developers, such as Vinu, this is important.

Reply:
I don't agree that it still should be the *only* way to run it.

Reply:
Okay, so what do you suggest for development then?

Reply:
I for example hook everything up with the IDE so that I get as optimized flow as possible. That is a lot harder with Docker and has extra delay. It is of course good that you can boot the other parts of the application as Docker container to automatize that.

Reply:
docker allows starting individual services. Would you like to me to add scripts wrapping these?

Reply:
I tried looking into a way to allow individual startup of dev-server of front-end and the dockerized way but unfortunately the proxy part is too tricky for me to deal with.
I suggest either we ditch the front-end from docker, or we use docker and modify npm run start script so that it launch a single "front-end" docker service instead of launch an actual react dev server